From a8fb1d8fba1e9557a3370edd62501f0a866c6774 Mon Sep 17 00:00:00 2001
From: Daiki Ueno <ueno@unixuser.org>
Date: Fri, 02 Nov 2012 06:19:47 +0000
Subject: Fix border-image slice values in antler's gtk CSS

According to CSS spec, border-image slice values are not denoted
with length units.

https://bugzilla.gnome.org/show_bug.cgi?id=687409
---
diff --git a/data/antler/style.css b/data/antler/style.css
index 0e830dc..7553cf6 100644
--- a/data/antler/style.css
+++ b/data/antler/style.css
@@ -12,7 +12,7 @@
 .antler-keyboard-button {
   border-width: 0px;
   border-radius: 2px;
-  border-image: url("dark-key-border.svg") 2px 2px 2px 2px repeat stretch;
+  border-image: url("dark-key-border.svg") 2 2 2 2 repeat stretch;
   font: Sans 14px;
   background-image: -gtk-gradient (linear,
 				     left top,
--
cgit v0.9.0.2
